src/pages/machine-learning/neural-networks/index.md

```diff
@@ -31,3 +31,81 @@ Neural networks are used on a variety of tasks. These include computer vision, s
 - <a href='http://ufldl.stanford.edu/tutorial/supervised/MultiLayerNeuralNetworks/' target='_blank' rel='nofollow'>Stanford University</a>
 - <a href='https://youtu.be/aircAruvnKk' target='_blank' rel='nofollow'>3Blue1Brown</a>
 - <a href='https://youtu.be/h3l4qz76JhQ' target='_blank' rel='nofollow'>Siraj Raval</a>
+
+My Changes  -----------------
```

Comment:
"My Changes" should not be in this article. Same with "End". Your submission should be in the form that the final article should take.